coreTable/CoreTableUtils.ts

```diff
@@ -227,7 +228,8 @@ export const makeKeyFn = (
     columnStore: CoreColumnStore,
     columnSlugs: ColumnSlug[]
 ) => (rowIndex: number) =>
-    columnSlugs.map((slug) => columnStore[slug][rowIndex].toString()).join(" ")
+    // toString() handles `undefined` and `null` values, which can be in the table.
```

Comment:
Makes sense. Relatedly I think it'd be ideal to keep undefineds and nulls out of the table at some point, but haven't nailed that down yet (and not entirely sure it's important).